streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueStore.java

```diff
@@ -106,6 +108,8 @@ public void putAll(final List<KeyValue<Bytes, byte[]>> entries) {
 
     @Override
     public <PS extends Serializer<P>, P> KeyValueIterator<Bytes, byte[]> prefixScan(final P prefix, final PS prefixKeySerializer) {
+        Objects.requireNonNull(prefix);
+        Objects.requireNonNull(prefixKeySerializer);
```

Comment:
Should we not check for `null` in the outer `MeteredXxxStore`? Should be redundant to check here for a second time.